staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -56,28 +56,12 @@ const (
 	minRetryDelay = 5 * time.Second
 	maxRetryDelay = 300 * time.Second
 
-	// labelNodeRoleMaster specifies that a node is a master. The use of this label within the
-	// controller is deprecated and only considered when the LegacyNodeRoleBehavior feature gate
-	// is on.
-	labelNodeRoleMaster = "node-role.kubernetes.io/master"
-
 	// labelNodeRoleExcludeBalancer specifies that the node should not be considered as a target
 	// for external load-balancers which use nodes as a second hop (e.g. many cloud LBs which only
 	// understand nodes). For services that use externalTrafficPolicy=Local, this may mean that
 	// any backends on excluded nodes are not reachable by those external load-balancers.
-	// Implementations of this exclusion may vary based on provider. This label is honored starting
-	// in 1.16 when the ServiceNodeExclusion gate is on.
+	// Implementations of this exclusion may vary based on provider.
 	labelNodeRoleExcludeBalancer = "node.kubernetes.io/exclude-from-external-load-balancers"
```

Comment:
followup nit - we should rename this to "labelNodeExcludeBalancers" or something  -  "role" has no place any more :)

Reply:
want me to open a new issue and mark as a good first issue? I'm on it :D

Reply:
https://github.com/kubernetes/kubernetes/issues/97734 :)